extensions/cdc-postgres/src/test/java/com/hazelcast/jet/cdc/postgres/MultiTableCacheIntegrationTest.java

```diff
@@ -256,16 +202,14 @@ private static class CustomerEntryProcessor implements EntryProcessor<Integer, O
         @Override
         public Object process(Entry<Integer, OrdersOfCustomer> entry) {
             try {
-                boolean deletion = Operation.DELETE.equals(record.operation());
                 OrdersOfCustomer value = entry.getValue();
-                if (deletion) {
-                    if (value != null) {
-                        value.setCustomer(null);
-                    }
+                if (value == null && (record.operation() == SYNC || record.operation() == INSERT)) {
+                    value = new OrdersOfCustomer();
+                }
+                requireNonNull(value, "value is null");
+                if (DELETE == record.operation()) {
+                    value.setCustomer(null);
```

Comment:
Wouldn't it make more sense to make the `value = null` here and get rid of the entry 1005 in the expected map?

Maybe the current approach is intentional, IDK :)

Reply:
From my understanding of the test, here we are checking that data was created and then cleared, rather than removed completely. We can track that INSERT -> DELETE track was processed correctly

Reply:
OK :+1: